### ISSUE TITLE
nlmixr2::nlmixr2 for SAEM - implement live traceplot (as done for Monolix)

### ISSUE THREAD

**Issue by iamstein** [nlmixr2/nlmixr2#167]
Monolix offers a live traceplot graphic while SAEM is running.  This is nice because it gives you an early indicator if your run is having convergence issues and if so, what parameters are causing the problem.  This way, instead of waiting for all the iterations to complete before recognizing the issue, you might be able to spot it early on, stop the run, and fix the issue.

Here's a video at 6:35, though they sped up the run time considerably in the video.
https://youtu.be/CyhstNWJeJY?t=395

A possible interim fix would be to allow the user to set the width of the Fit Data object to a very large value so at least, one could try and look at the numbers and they'd all line up well.  For instance, when there are a lot of parameters, the output might look like this:
![image](https://github.com/nlmixr2/nlmixr2/assets/2180507/4882ec15-c1bf-43f3-8c89-7c4170d294ac)

but it's much more readable if it's one row per iteration and I could use the left-right scroll bars if needed.  though it'd be nice too if the parameters and columns lined up.
![image](https://github.com/nlmixr2/nlmixr2/assets/2180507/37ab7718-0a9b-459d-9aa0-865c8eb6d892)

**Comment by mattfidler** [comment]
I agree.  I also think adding iterations to a phase would be nice like Monolix, though that may be a bit more work 😄
